### ISSUE TITLE
Restore HWNAT function

### ISSUE THREAD

**Issue by quanah** [ILOVEPIE/qts1kpcie_wifi_driver#5]
As noted in a comment on January of 2018, the HW NAT code was removed from this driver and needs to be restored.  Opening an issue specific to that work.

**Comment by ILOVEPIE** [comment]
Yeah the HWNAT functions were removed in error they do need to be restored, haven't had the time to do so, if you want to tackle that i'd be greatful, i've been kinda busy.

**Comment by quanah** [comment]
I wish I knew what that entailed. :) I just saw the reference in another issue and thought it deserved its own tracker item. ;)

I was going to rebuild my netgear nighthawk r7500 with the driver here but wanted the HW NAT functionality as well.

**Comment by ILOVEPIE** [comment]
I'd suggest reading through the code and then looking at how OpenWRT does hardware nat and integrating it in.
